portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -36,15 +37,16 @@ class ProjectControllerItTest
   "ProjectController" when {
     "getProjectByName" should {
       "return a project with the same name" in {
-        val dummyProject = TestProjectUtils.getDummyProject(ownerId = dummyUser.userId)
+        val dummyProject =
+          TestProjectUtils.getDummyProject(ProjectId(UserId.random.unwrap), dummyUser.userId)
```

Comment:
`ProjectId` is created using `UserId`?:) It looks strange.

Reply:
Done